integration_tests/integration/recommendation.spec.ts

```diff
@@ -2266,6 +2266,29 @@ context('Make a recommendation', () => {
       cy.pageHeading().should('contain', 'Edit custody type')
     })
 
+    it('edit Current Establishment', () => {
+      cy.task('getRecommendation', {
+        statusCode: 200,
+        response: {
+          ...completeRecommendationResponse,
+          prisonOffender: {},
+          bookRecallToPpud: {
+            currentEstablishment: 'The Kyln',
+          },
+          ppudOffender: {},
+        },
+      })
+      cy.task('getStatuses', { statusCode: 200, response: [{ name: 'SENT_TO_PPCS', active: true }] })
```

Comment:
I have seen this `'SENT_TO_PCS'` status appearing in a bunch of places. Should this be a const somewhere?

Reply:
Good point! It is already a const somewhere, I'll update this and the rest of the similar cases in this source file.

Reply:
The project wouldn't build for some reason I couldn't pinpoint when importing said enum from `recommendationStatusCheck.ts`. Since the enum isn't used within that source, I've made a copy of it into a separate source and imported that, which worked (which suggests an issue with what else it pulls in form `recommendationStatusCheck.ts` when doing the import, but I couldn't find out the cause). I've left the old enum around too, as it's referenced by too many files to move over with this PR, but have opened https://dsdmoj.atlassian.net/browse/MRD-2688 for this.